### PR TITLE
Adds collection of firmware elements from lxca

### PR DESCRIPTION
**Pull request by rodneyhbrown7**
@miq-bot add_label: wip, providers/physical-infrastructure

### CODE REVIEW

**Comment by juliancheal** [comment]
@miq-bot add_label wip, providers/physical-infrastructure

**Comment by rodneyhbrown7** [comment]
@miq-bot remove_label wip

File: app/models/manageiq/providers/lenovo/physical_infra_manager/physical_server.rb [hunk 1]
```diff
@@ -3,5 +3,7 @@ class Lenovo::PhysicalInfraManager::PhysicalServer < ::PhysicalServer
     def name
       "physical_server"
     end
+
+    has_many :firmwares, :foreign_key => "ph_server_uuid", :primary_key => "uuid"
```
Comment (juliancheal) [inline app/models/manageiq/providers/lenovo/physical_infra_manager/physical_server.rb]: This `has_many` line isn't needed.

**Comment by juliancheal** [comment]
LGTM 👍 Just remove that one line in my previous comment. :)

**Comment by rodneyhbrown7** [comment]
@miq_bot add_label providers/physical_infrastructure

**Comment by juliancheal** [comment]
> ⚠️ - Line 63, Col 7 - Lint/UselessAssignment - Useless assignment to variable - new_result.

I kinda like having a variable in a method that just returns a hash. Sorry Rubocop.

**Pull request merged**